_includes/community/community-members.html

```diff
@@ -1,29 +1,49 @@
 <h3 class="black-text">Our Community Members</h3>
 <img
 src="/assets/images/programs/Layer5-MeshMates.png"
-width="60%" align="right" style="margin:5px;"
+width="40%" align="right" style="margin:5px;"
 />
 <div>
   <h6>Jump in! The community is warm</h6>
-  With members across the world, the Layer5 community is diverse. Layer5 community members are individuals who have shown consistent commitment to advancing Layer5 open source initiatives.
-
+  The Layer5 community is diverse with members from around the globe, who have shown consistent commitment in
+  advancing our open source initiatives.
   <p class="indent">
-    We believe that all contributors should expect and be part of a safe and
-    friendly environment for constructive contribution. We can more effectively
-    and successfully compare and challenge different ideas to find the best
-    solutions for advancement, while building the size, diversity, and strength
-    of our community.
-  </p>
-  <div width="100%" style="justify-content:center; text-align: center;">
+    We believe that every contributor deserves a safe and friendly environment, enabling them to effectively compare different ideas and find the best solutions for advancement, while building the size, diversity, and strength of the community.
```

Comment:
We'll want to enforce a column wrap of 80.

You can use VS Code's "Prettier - Code formatter" plugin to achieve this.

Reply:
It's not critical, but text isn't wrapped here.